purchase_deposit/wizard/purchase_advance_return_option.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2019 Elico Corp, Dominique K. <dominique.k@elico-corp.com.sg>
+# Copyright 2019 Ecosoft Co., Ltd., Kitti U. <kittiu@ecosoft.co.th>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class PurchaseAdvanceReturnOption(models.TransientModel):
+    _name = "purchase.advance.return.option"
+    _description = "Select how to return advance in vendor bill"
+
+    advance_return_option = fields.Selection(
+        [
+            ("proportional", "Propotional Return"),
```

Comment:
I think `Clear` is a simpler term to understand than `Return`

What do you think?

Reply:
I think return is also good. May I keep it this way?